doc/configuration.md

```diff
@@ -17,6 +17,7 @@ The following table describes standard paths used by the runtime:
 | `ENROOT_DATA_PATH` | `${XDG_DATA_HOME}/enroot` | Path to user container storage |
 | `ENROOT_TEMP_PATH` | `${TMPDIR}` | Path to temporary directory |
 
+If there is a directory `enroot.conf.d` next to `enroot.conf`, all files in that directory matching `*.conf` will be considered, too.
```

Comment:
Hey, great PR :1st_place_medal: 

maybe this should also say that the `enroot.conf` is read first and then the `enroot.conf.d`. 

Example:

If there are both `enroot.conf` and `enroot.conf.d`, the file `enroot.conf` is read first and then all files the directory matching `*.conf` (read in shell expansion order).

Reply:
Thanks for the idea